From 80ee5632d92a861ca36acd2da6c4ce4f006cb8ba Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Fri, 7 Sep 2018 13:51:39 -0500
Subject: [PATCH] add net.ipv4.ip_forward = 1 sysctl conf file

When you create a cluster in K8s, you need to set ip_forward.

https://kubernetes.io/docs/setup/scratch/#network-connectivity
https://kubernetes.io/docs/concepts/cluster-administration/networking/

enable it only when you install Kubernetes.
---
 60-k8s-sysctl.conf | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 60-k8s-sysctl.conf

diff --git a/60-k8s-sysctl.conf b/60-k8s-sysctl.conf
new file mode 100644
index 00000000..01b0f59d
--- /dev/null
+++ b/60-k8s-sysctl.conf
@@ -0,0 +1,4 @@
+# IP forward is need to create a K8s cluster
+# https://kubernetes.io/docs/setup/scratch/#network-connectivity
+# https://kubernetes.io/docs/concepts/cluster-administration/networking/
+net.ipv4.ip_forward = 1
-- 
2.18.0

